src/__tests__/functions.test.js

```diff
@@ -0,0 +1,13 @@
+import { fahreinheitToCelcius, getHoursAndMinutes } from '../utils/functions';
+
+describe('Function tests', () => {
+  it('should be equal to ', () => {
+    const date = getHoursAndMinutes(1673041519);
+    expect(date).toEqual('10:45 PM');
```

Comment:
- [ ] This test is failing kindly make sure to fix it so it passes

![Screenshot_20230107_001219](https://user-images.githubusercontent.com/25227022/211115992-e8870b51-319d-455e-ad11-e0ab28d04b54.png)

Reply:
![image](https://user-images.githubusercontent.com/50754458/211121166-d00f9819-ce67-4ba3-806e-a7f753a3d7f5.png)
When I changed it to your time, it's not working on my side, It depends on the timezone